examples/java/src/main/java/org/apache/beam/examples/DebuggingWordCount.java

```diff
@@ -62,12 +62,14 @@
  * }
  * </pre>
  *
- * <p>To execute this pipeline using the Dataflow service and the additional logging discussed
- * below, specify pipeline configuration:
+ * <p>To change the runner, specify:
+ * <pre>{@code
+ *   --runner=YOUR_SELECTED_RUNNER
+ * }
+ * </pre>
+ *
+ * <p>To use the additional logging discussed below, specify:
```

Comment:
Can you file a JIRA in the documentation component? (or the right one, if there is such a thing)